content/spin/v2/writing-apps.md

```diff
@@ -211,6 +211,14 @@ HTTP path: /...
 
 {{ startTab "Python"}}
 
+Ensure that you have Python 3.10 or later installed on your system. You can check your Python version by running:
```

Comment:
This isn't particularly associated with the template - someone picking up an existing application to maintain might never use the template but would still need this info.  To me, it fits more naturally in the "Setting up for `spin build`" section on the build page, which is where we talk about required tools and tool configuration.

It might also be good to mention it in a build troubleshooting section.